Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM docker.io/fedora:31
+
+RUN dnf update -y \
+ && dnf install -y \
+        direnv \
+        findutils \
+        git \
+        python3-virtualenv \
+        unzip \
+        which \
```

Comment:
These packages are a bit arbitrary. Some of them aren't always necessary but are commonly used. I guess we can start with things we use, add things as we go, and decide to deal with it later if it gets too big.